src/back/models/attributes/Attribute.js

```diff
@@ -222,6 +225,11 @@ function Attribute() {
       'Invalid property "name" when creating an Attribute (it has to be a ' +
       'string)'
     );
+    expect(_reservedNames).to.not.include(
+      attribute.name,
+      'Invalid property "name" when creating an Attribute (it must not ' +
```

Comment:
I would complain about your first comment, but since the second comment was extremely useful, I owed you one. So "thanks". :octocat: